app/controllers/find/feature_flags_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Find
+  class FeatureFlagsController < ApplicationController
+    before_action :enforce_basic_auth
+    skip_before_action :redirect_to_maintenance_page_if_flag_is_active
+
+    def index; end
+
+    def update
+      FeatureFlag.send(action, feature_name)
+
+      if Rails.env.production?
+        SlackNotificationJob.perform_now(
+          ":flags: Feature ‘#{feature_name}‘ was #{action}d",
+          find_feature_flags_path,
+        )
+      end
+
+      flash[:success] = "Feature ‘#{feature_name.humanize}’ #{action}d"
```

Comment:
This don't get displayed in the right place, regression.

Reply:
Fixed

<img width="1065" alt="Screenshot 2022-11-09 at 14 32 10" src="https://user-images.githubusercontent.com/616080/200857397-d89f9144-e84c-4b5b-ad27-2200f61bed91.png">